src/psyclone/psyGen.py

```diff
@@ -1766,14 +1765,6 @@ def rename_and_write(self):
             new_kern_code = fortran_writer(self.get_kernel_schedule().root)
```

Comment:
Sorry to be a pain but I think this needs a bit more tidying - AFAICS, `self._kern_schedule` isn't actually used directly as we do `self.get_kernel_schedule().root`. Therefore, please could you remove the `if` at line 1752 and un-indent the corresponding block.

Reply:
Done